Calc.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by User on 01.10.2016.
+ */
+public class Calc {
+
+    public static void main(String[] args) {
+
+        System.out.println("Ur numbers r: " + Integer.parseInt(args[0]) + " " + Integer.parseInt(args[1]));
+        int a = Integer.parseInt(args[0]);
+        int b = Integer.parseInt(args[1]);
+        System.out.println("sum: " + (a + b) + " " + "proizvidenie: " + (a * b) + " " + "raznost: " + (a - b) + " " + "delenie: " + (a / b));

```

Comment:
if user tries to run application with 0 and 0 it will print special values for double or stop with exception for integers